JitsiConference.js

```diff
@@ -1370,19 +1363,20 @@ JitsiConference.prototype.onMemberJoined = function(
 /**
  * Updates features for a participant.
  * @param {JitsiParticipant} participant - The participant to query for features.
- * @returns {Promise<Set<String> | never>}
+ * @returns {void}
  * @private
  */
 JitsiConference.prototype._updateFeatures = function(participant) {
-    return this.xmpp.caps.getFeatures(participant.getJid())
+    participant.getFeatures()
         .then(features => {
             participant._supportsDTMF = features.has('urn:xmpp:jingle:dtmf:0');
             this.updateDTMFSupport();
 
             if (features.has('http://jitsi.org/protocol/jigasi')) {
                 participant.setProperty('features_jigasi', true);
             }
-        });
+        })
+        .catch(() => false);
```

Comment:
Hum, in case of a mismatch there will be print, but in the case of error stanza or timeout, there will be none. Yep, I will add a print, I was thinking only for the first case.

Reply:
Just checked the diff, the getFeatures from JitsiParticipant always prints an error if there is one, so nothing to log here.